activerecord/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
+*   OpenSSL constants are now used for Digest computations.
+
+    *Dirkjan Bussink*
+
 *   Relation#destroy_all perform its work in batches
 
+*   Prevent double saves in autosave of cyclic associations
```

Comment:
I think some rebase conflict got added in changelog. The description below is for changes related to `Relation#destroy_all` in batches.

Reply:
Thanks! Fixed in ce7ff88a50c31d4ccea08a63ef8f698c5d25ba36

Reply:
@eileencodes https://github.com/rails/rails/commit/ce7ff88a50c31d4ccea08a63ef8f698c5d25ba36 removed wrong changelog. 

We need to keep this line `* Relation#destroy_all perform its work in batches` and remove `* Prevent double saves in autosave of cyclic associations` 😊